R/pkg/R/functions.R

```diff
@@ -2159,6 +2162,9 @@ setMethod("to_json", signature(x = "Column"),
 #' @details
 #' \code{to_csv}: Converts a column containing a \code{structType} into a Column of CSV string.
 #' Resolving the Column can fail if an unsupported type is encountered.
+#' You can find the CSV-specific options for writing CSV files in
+#' \href{https://spark.apache.org/docs/latest/sql-data-sources-csv.html#data-source-option}{
+#' Data Source Option} in the version you use.
```

Comment:
You will have to remove it here, and fix the documentation above instead. See https://github.com/apache/spark/blob/45ba5a63b35e2e6e501d6288cca8929c773adc45/R/pkg/R/functions.R#L256-L268

Reply:
Oh, I see. let me fix it. Thanks!